Command/ClearMetadataCacheDoctrineODMCommand.php

```diff
@@ -11,11 +11,12 @@
 
 /**
  * Command to clear the metadata cache of the various cache drivers.
+ *
+ * @internal since 4.7
```

Comment:
```suggestion
 * @internal
```
This applies to some `@internal` in the PR

It would be nice to see if we can remove these Proxy commands (in another PR), ref https://github.com/doctrine/DoctrineBundle/issues/1524

Reply:
Fixed by https://github.com/doctrine/DoctrineMongoDBBundle/pull/800